pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/route/familymember/AddFamilyMemberRouteTest.java

```diff
@@ -45,7 +45,7 @@ public static void doFirst() {
         setupDB();
         gson = new Gson();
         String suffix = System.currentTimeMillis() + "";
-        ddpInstanceDto = DBTestUtil.createTestDdpInstance(ddpInstanceDto, ddpInstanceDao, "AddFamilyMemberInstance" + suffix);
+        ddpInstanceDto = DBTestUtil.createTestDdpInstance(ddpInstanceDao, "AddFamilyMemberInstance" + suffix);
```

Comment:
Is the @Ignore on this class intentional?

Reply:
So updated to compile but still broken?

Reply:
Correct.